runner/src/server/plugins/logger.ts

```diff
@@ -0,0 +1,5 @@
+import pino from "pino";
+
+export default pino({
+  name: "designer",
+});
```

Comment:
please don't use this! we already register pino to the server. This configuration is also incorrect "designer"

see `runner/src/server/plugins/logging.ts`

Reply:
Thanks, well spotted on the "designer/runner" mistake. 

To clarify the reason for this, there are some scenarios where we *don't* have a request or server available so we cannot use hapi-pino.

eg. runner/src/server/plugins/engine/services/addressService.ts

Would you rather we just did a console.log/error here instead?